web/styles/message_row.css

```diff
@@ -1068,6 +1068,58 @@ of the base style defined for a read-only textarea in dark mode. */
 .bookend {
     margin-top: 10px;
     background-color: transparent;
+    padding-bottom: 10px;
+    text-align: center;
+
+    .stream-status {
+        opacity: 0.6;
+
+        & i {
+            font-size: 0.7143em; /* 10px at 14px / em */
+            /* TODO: Don't use relative positioning here. */
+            position: relative;
+            top: 0.1em; /* 1px at 10px / em */
+            margin-left: 0.2em; /* 2px at 10px / em */
+        }
+
+        .zulip-icon.zulip-icon-lock {
+            font-size: 0.8571em; /* 12px at 14px / em */
+            top: 0.0833em; /* 1px at 12px / em */
+            margin-left: 0.1667em; /* 2px at 12px / em */
+        }
+    }
+}
+
+.bookend-label {
+    font-size: 1em;
+    display: block;
+    padding: 4px;
+    overflow: hidden;
+
+    &::after,
+    &::before {
+        content: "";
+        display: inline-block;
+        position: relative;
+        vertical-align: middle;
+        width: 50%;
+        height: 0;
+        border-bottom: 1px solid
+            light-dark(
+                color-mix(in oklch, hsl(0deg 0% 0%) 15%, transparent),
```

Comment:
Could we just make these colors CSS variables?

Reply:
Used a variable called `--color-border-mesaage-group-spacer-line` to define this spacer line color.